app/Resources/views/line/bundle_card.html.twig

```diff
@@ -1,47 +1,62 @@
-            </h2>
-            <div class="bundle-card-line-description">
-                {{ ez_render_field(content, 'description') }}
+<div class="bundle-card-line-container{% if extra.page is defined %} page-{{ extra.page }}{% endif %}">
+    <div class="bundle-card-line">
+        <div class="bundle-card-line-data">
+            <div class="bundle-card-line-information">
+                <h1>
+                    {{ ez_render_field(content, 'packagist_url') }}
+                </h1>
+                <div class="bundle-card-line-description">
+                    {{ ez_render_field(content, 'description') }}
+                </div>
             </div>
-        </div>
-        <div class="col-sm-4 col-md-3 col-lg-2 bundle-card-line-author">
-            <div class="bundle-card-line-author-container">
-                {% if packageDetails.author[0] is defined %}
-                    <div class="bundle-card-line-author-username">{{ packageDetails.author[0].name }}</div>
-                {% endif %}
-                {% if packageDetails.authorAvatarUrl is defined and packageDetails.author[0].name is defined %}
-                    <div class="bundle-card-line-author-photo">
-                        <img src="{{ packageDetails.authorAvatarUrl }}?s=100" alt="{{ packageDetails.author[0].name }}" />
-                    </div>
-                {% endif %}
-                {% if packageDetails.updated is defined %}
-                    <div class="bundle-card-line-last-update">Updated: <span>{{ packageDetails.updated|date("d / m / Y") }}</span></div>
-                {% endif %}
+            <div class="bundle-card-line-counters">
+                <ul>
+                    <li class="downloads"><i class="fa fa-cloud-download" aria-hidden="true"></i> {{ ez_field_value(content, 'downloads') }}</li>
+                    <li class="stars"><i class="fa fa-star" aria-hidden="true"></i> {{ ez_field_value(content, 'stars') }}</li>
+                    <li class="forks"><i class="fa fa-code-fork" aria-hidden="true"></i> {{ ez_field_value(content, 'forks') }}</li>
+                    <li class="updated"><i class="fa fa-clock-o" aria-hidden="true"></i> {{ ez_field_value(content, 'updated')|date("d / m / Y") }}</li>
+                </ul>
             </div>
-        </div>
-    </div>
-    {% if packageDetails.maintainers is defined %}
-        <div class="hidden-xs col-sm-12 bundle-card-line-contributors">
-            <div class="bundle-card-line-contributors-container">
-                <span>By:</span>
-                    {% for maintainer in packageDetails.maintainers|slice(0, 4) %}
-                        <div class="bundle-card-line-contributor">
-                            <div class="bundle-card-line-contributor-photo">
+            <div class="bundle-card-line-authors">
+                <div class="bundle-card-line-owner">
+                    {% if packageDetails.authorAvatarUrl is defined %}
```

Comment:
Why not having the profile picture and the name of the Owner under the same `if` condition?
If something happens and you don't have the picture, the page will display only the name and not `<span>Owner:</span>`